app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split(" ")
+    mv = command[0]
+    file_to_move = command[1]
+    destination = command[2].split("/")
+    if mv == "mv" and file_to_move != "".join(destination):
+        try:
+            with open(file_to_move, "r") as f:
+                data = f.read()
+            os.remove(file_to_move)
```

Comment:
Code Style issue: The variable name 'f' is not descriptive. You should use a more descriptive name for your variables. For example, you could use 'source_file' instead of 'f'.